packages/buttons/src/views/icon-button/Icon.spec.js

```diff
@@ -29,4 +29,14 @@ describe('Icon', () => {
 
     expect(container.firstChild).toHaveClass('is-rotated');
   });
+
+  it('passes className prop', () => {
+    const { container } = render(
+      <Icon className="test">
+        <svg />
+      </Icon>
+    );
+
+    expect(container.firstChild.classList[1]).toBe('test');
```

Comment:
We have a jest matcher to help with these class assertions. Maybe give this a try:

```suggestion
    expect(container.firstChild).toHaveClass('test');
```